source/_integrations/ultrasync.markdown

```diff
@@ -0,0 +1,88 @@
+---
+title: Interlogix UltraSync Hub
+description: Instructions on how to integrate your Interlogix UltraSync Hub within Home Assistant.
+ha_category:
+  - Alarm
+logo: ultrasync.png
+ha_iot_class: Local Polling
+ha_release: 0.119
+ha_config_flow: true
+ha_codeowners:
+  - '@caronc'
+ha_domain: ultrasync
+---
+
+The `ultrasync` platform will allow you to monitor and control your Interlogix UltraSync Hub from within Home Assistant and setup automation.
+
+## Configuration
+
+Go to the integrations page in your configuration and click on new integration -> UltraSync.
+
+**Note**: You can only be logged into the ZeroWire/UltraSync hub with the same user once; a subsequent login with the same user logs out the other. Since Home Assistant (HA) actively polls and maintains a login session to this Hub, it can prevent you from being able to log into at the same time elsewhere (via it's website). It is strongly recommended that you create a second user account on your Hub dedicated for just HA.
+
+## Sensor
+
+This component will create these sensors in the format of `{ultrasync_hubname}_{sensor}`;  The below example assumes you accept the default name of `UltraSync` (which is still represented in lowercase):
```

Comment:
The names entity names can be changed by the user, right? They are also visible in the device overview, right?

So the device entity name hint should not be needed.